src/hotspots/HotspotTransform.js

```diff
@@ -2,116 +2,182 @@
+
+        rotation.x = (typeof config.rotation.x === "number") ? config.rotation.x : 0;
+        rotation.y = (typeof config.rotation.y === "number") ? config.rotation.y : 0;
+        rotation.z = (typeof config.rotation.z === "number") ? config.rotation.z : 0;
+
+        if(FORGE.Utils.compareObjects(this._rotation.dump(), rotation) === false)
         {
-            var radius = (typeof config.position.radius === "number") ? config.position.radius : 200;
-            var theta = (typeof config.position.theta === "number") ? FORGE.Math.degToRad(config.position.theta) : 0;
-            var phi = (typeof config.position.phi === "number") ? FORGE.Math.degToRad(config.position.phi) : 0;
+            this._rotation.load(/** @type {HotspotTransformValuesConfig} */ (rotation), false);
+            changed = true;
+        }
+    }
 
-            theta = FORGE.Math.wrap(Math.PI - theta, -Math.PI, Math.PI);
+    if (typeof config.scale !== "undefined")
+    {
+        var scale = FORGE.Utils.extendSimpleObject({}, this._scale.dump());
 
-            var cartesian = new THREE.Vector3().setFromSpherical(FORGE.Utils.toTHREESpherical(radius, theta, phi));
+        scale.x = (typeof config.scale.x === "number") ? FORGE.Math.clamp(config.scale.x, 0.000001, 100000) : 1;
+        scale.y = (typeof config.scale.y === "number") ? FORGE.Math.clamp(config.scale.y, 0.000001, 100000) : 1;
+        scale.z = (typeof config.scale.z === "number") ? FORGE.Math.clamp(config.scale.z, 0.000001, 100000) : 1;
 
-            this._position.x = cartesian.x;
-            this._position.y = cartesian.y;
-            this._position.z = cartesian.z;
+        if(FORGE.Utils.compareObjects(this._scale.dump(), scale) === false)
+        {
+            this._scale.load(/** @type {HotspotTransformValuesConfig} */ (scale), false);
+            changed = true;
         }
     }
 
-    if (typeof config.rotation !== "undefined")
+    return changed;
+};
+
+/**
+ * Parse the position object.
+ * @method FORGE.HotspotTransform#_parsePosition
+ * @param {HotspotTransformPosition} config - The transform position config to parse.
+ * @private
+ */
+FORGE.HotspotTransform.prototype._parsePosition = function(config)
+{
+    var position =
     {
-        this._rotation.x = (typeof config.rotation.x === "number") ? config.rotation.x : 0;
-        this._rotation.y = (typeof config.rotation.y === "number") ? config.rotation.y : 0;
-        this._rotation.z = (typeof config.rotation.z === "number") ? config.rotation.z : 0;
+        x: 0,
+        y: 0,
+        z: -200
+    };
+
+    if (typeof config !== "undefined" && config !== null)
+    {
+        position.x = (typeof config.x === "number") ? config.x : 0;
+        position.y = (typeof config.y === "number") ? config.y : 0;
+        position.z = (typeof config.z === "number") ? config.z : -200;
+
+        if(typeof config.radius === "number" || typeof config.theta === "number" || typeof config.phi === "number")
```

Comment:
`radius/theta/phi` were marked as "soon to be deprecated" in the JSON reference.  Do we remove this comment then ? Because if we follow the system here, the `radius/theta/phi` is getting the upper hand on `x/y/z` (in configuration reading, not stored value). So it may not be a bad idea to specify it in the JSON ref.